server/api/swagger/user.yaml

```diff
@@ -921,6 +943,7 @@ paths:
           description: CSV
           schema:
             type: string
+            maxLength: 256
```

Comment:
@fetok12 why did you put that in the response? This would block the csv I think.